lib/util.js

```diff
@@ -112,6 +112,23 @@ _.mixin(/** @lends util */ {
     },
 
     /**
+     * Merges defined keys from the target object onto the source object.
+     *
+     * @param {Object} target
+     * @param {Object} source
+     * @returns {Object}
+     */
+    mergeDefined: function (target, source) {
+        Object.getOwnPropertyNames(source).forEach(function (key) {
```

Comment:
No no. Simple for loop please. This is a core library function and should be performant. 

For var prop in source
If source has own property prop and prop is not undefined, …